tests/stress_tests/memcheck_tests/tests.cpp

```diff
@@ -106,3 +106,51 @@ INSTANTIATE_TEST_CASE_P(MemCheckTests, MemCheckTestSuite,
                         ::testing::ValuesIn(
                                 generateTestsParams({"devices", "models"})),
                         getTestCaseName);
+
+TEST_P(MemCheckTestSuite, inference_with_streams) {
+    const auto nstreams = 2;
+    log_info("Inference of InferRequest from network: \"" << model
+                                                          << "\" with precision: \"" << precision
+                                                          << "\" for device: \"" << device << "\""
+                                                          << "\" with streams: \"" << nstreams);
+    if ((device != "CPU") && (device != "GPU"))
+        throw std::invalid_argument("This device is not supported");
+
+    auto test_pipeline = [&] {
+        MemCheckPipeline memCheckPipeline;
+        Core ie;
+        std::map<std::string, std::string> config;
+        const std::string key = device + "_THROUGHPUT_STREAMS";
+        InferRequest inferRequest;
+
+        config[key] = std::to_string(nstreams);
+        ie.GetVersions(device);
```

Comment:
Do we need it here?

Reply:
It's a copy from other tests and it was required to trigger plugin loading to track it separately from other IE pipeline steps. But because we track memory only on pipeline finish, it's not a required step. Let's leave it as is to align with previous tests